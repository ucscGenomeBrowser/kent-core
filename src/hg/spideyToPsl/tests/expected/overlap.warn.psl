813	1	0	0	0	0	4	2471	++	BC017206_part	814	0	814	chr19.f0	4984	100	3385	5	62,382,190,95,85,	0,62,444,634,729,	100,836,1294,1744,3300,
