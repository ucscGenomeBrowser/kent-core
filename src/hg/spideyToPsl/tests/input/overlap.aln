--SPIDEY version 1.40--
Genomic: lcl|chr19.f0 No definition line found, 4984 bp
mRNA: lcl|BC017206 No definition line found, 1920 bp
Strand: plus
Number of exons: 9
Exon 1: 101-162 (gen)  1-62 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 1  0
Exon 2: 837-1218 (gen)  63-444 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 1  1
Exon 3: 1295-1484 (gen)  445-634 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 1  1
Exon 4: 1745-1839 (gen)  635-729 (mRNA)  id 98.9% mismatches 1 gaps 0  splice site (d  a): 1  1
Exon 5: 3301-3385 (gen)  730-814 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 1  1
Exon 6: 3588-3677 (gen)  815-904 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 1  1
Exon 7: 3834-4040 (gen)  905-1111 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 1  1
Exon 8: 4314-4338 (gen)  1112-1136 (mRNA)  id 96.0% mismatches 1 gaps 0  splice site (d  a): 0  1
Exon 9: 4330-4886 (gen)  1317-1873 (mRNA)  id 99.8% mismatches 1 gaps 0  splice site (d  a): 0  0
Number of splice sites: 7
mRNA coverage: 90%
overall percent identity: 99.8%
Missing mRNA ends: neither 
Non-aligning poly(A)+ tail length: 49

Genomic: lcl|chr19.f0 No definition line found
mRNA: lcl|BC017206 No definition line found
Exon 1: 101-162 (gen)  1-62 (mRNA)


GGGACAGATCCCGAAGTTCGAGCATCCCTCGGATAGGCCGGGTGTCAGGCCTGGTCTCTC
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          CCGAAGTTCGAGCATCCCTCGGATAGGCCGGGTGTCAGGCCTGGTCTCTC

AGGCCCGTCCAGGTGGGCGTCT
||||||||||||
AGGCCCGTCCAG

Exon 2: 837-1218 (gen)  63-444 (mRNA)


TTTTCTGTAGGCCCATCTTGACGATTCCAAGACCACCCCCTTGAGCAAGAATGCCATCCC
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          GCCCATCTTGACGATTCCAAGACCACCCCCTTGAGCAAGAATGCCATCCC
                                             A  R  M  P  S 


CTCTGGGTCCCCCATGCCTGCCCGTCATGGACCCAGAGACCACCCTTGAGGAGCCTGAGA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CTCTGGGTCCCCCATGCCTGCCCGTCATGGACCCAGAGACCACCCTTGAGGAGCCTGAGA
P  L  G  P  P  C  L  P  V  M  D  P  E  T  T  L  E  E  P  E 


CTGCCCGCCTCCGCTTCCGAGGGTTCTGCTACCAGGAGGTGGCAGGTCCCCGAGAAGCCC
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CTGCCCGCCTCCGCTTCCGAGGGTTCTGCTACCAGGAGGTGGCAGGTCCCCGAGAAGCCC
T  A  R  L  R  F  R  G  F  C  Y  Q  E  V  A  G  P  R  E  A 


TGGCCCGGCTGCGTGAGCTGTGTTGCCAGTGGCTGCAGCCTGAGGCACACTCCAAGGAGC
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
TGGCCCGGCTGCGTGAGCTGTGTTGCCAGTGGCTGCAGCCTGAGGCACACTCCAAGGAGC
L  A  R  L  R  E  L  C  C  Q  W  L  Q  P  E  A  H  S  K  E 


AGATGCTGGAGATGCTGGTGCTGGAGCAGTTCCTGGGCACACTGCCTCCCGAGATCCAGG
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
AGATGCTGGAGATGCTGGTGCTGGAGCAGTTCCTGGGCACACTGCCTCCCGAGATCCAGG
Q  M  L  E  M  L  V  L  E  Q  F  L  G  T  L  P  P  E  I  Q 


CCTGGGTGCGCGGTCAGCGGCCAGGCAGTCCTGAGGAGGCCGCTGCCCTAGTCGAAGGAC
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CCTGGGTGCGCGGTCAGCGGCCAGGCAGTCCTGAGGAGGCCGCTGCCCTAGTCGAAGGAC
A  W  V  R  G  Q  R  P  G  S  P  E  E  A  A  A  L  V  E  G 


TGCAGCATGACCCTGGGCAACTGTTGGGCTGGGTGAGTGTGG
||||||||||||||||||||||||||||||||
TGCAGCATGACCCTGGGCAACTGTTGGGCTGG
L  Q  H  D  P  G  Q  L  L  G  W 


Exon 3: 1295-1484 (gen)  445-634 (mRNA)


TTCTCCTCAGATCACAGCCCATGTCCTGAAGCAGGAGGTGCTCCCTGCAGCCCAGAAGAC
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          ATCACAGCCCATGTCCTGAAGCAGGAGGTGCTCCCTGCAGCCCAGAAGAC
           I  T  A  H  V  L  K  Q  E  V  L  P  A  A  Q  K  T 


AGAGGAACCACTTGGGAGCCCCCACCCCTCAGGGACAGTGGAGTCCCCTGGGGAAGGTCC
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
AGAGGAACCACTTGGGAGCCCCCACCCCTCAGGGACAGTGGAGTCCCCTGGGGAAGGTCC
  E  E  P  L  G  S  P  H  P  S  G  T  V  E  S  P  G  E  G  P 


CCAGGACACCAGAATAGAGGGGTCTGTCCAGCTCAGCTGCAGTGTGAAGGAGGAGCCCAA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CCAGGACACCAGAATAGAGGGGTCTGTCCAGCTCAGCTGCAGTGTGAAGGAGGAGCCCAA
  Q  D  T  R  I  E  G  S  V  Q  L  S  C  S  V  K  E  E  P  N 


TGTCGATGGACAGGAAGTGGGTGAGGTTGG
||||||||||||||||||||
TGTCGATGGACAGGAAGTGG
  V  D  G  Q  E  V 


Exon 4: 1745-1839 (gen)  635-729 (mRNA)


CACTTTTCAGCACCCTCCAGCCCTCCACTTGCAGCACAGTCCCCTGAGGGGAACCATGGA
          ||||||||||||||||||||||||||||||||||||||||| ||||||||
          CACCCTCCAGCCCTCCACTTGCAGCACAGTCCCCTGAGGGGCACCATGGA
          A  P  S  S  P  P  L  A  A  Q  S  P  E  G  H  H  G 


CACCAAGAACCAGCCTCCACATCCTTCCACCCACCCAGGATTCAGGTGAGCAGCC
|||||||||||||||||||||||||||||||||||||||||||||
CACCAAGAACCAGCCTCCACATCCTTCCACCCACCCAGGATTCAG
 H  Q  E  P  A  S  T  S  F  H  P  P  R  I  Q 


Exon 5: 3301-3385 (gen)  730-814 (mRNA)


ACATCCGCAGGAGGAGTGGGGGCTGCTGGACCGGTCACAGAAGGAACTGTACTGGGATGC
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          GAGGAGTGGGGGCTGCTGGACCGGTCACAGAAGGAACTGTACTGGGATGC
           E  E  W  G  L  L  D  R  S  Q  K  E  L  Y  W  D  A 


GATGCTGGAGAAGTACGGCACAGTGGTCTCCCTGGGTGAGGACCA
|||||||||||||||||||||||||||||||||||
GATGCTGGAGAAGTACGGCACAGTGGTCTCCCTGG
  M  L  E  K  Y  G  T  V  V  S  L 


Exon 6: 3588-3677 (gen)  815-904 (mRNA)


ACCCGGGCAGGGTTACCGCCCCACCAGCCAGAGGCACAGGCCCAGTCAGAGCTGGGGATG
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          GGTTACCGCCCCACCAGCCAGAGGCACAGGCCCAGTCAGAGCTGGGGATG
          G  L  P  P  H  Q  P  E  A  Q  A  Q  S  E  L  G  M 


CTGCTCACGGGGACAGGCGTCTGCAGAAGCCTGCGCTCGGGTGAGTGCCC
||||||||||||||||||||||||||||||||||||||||
CTGCTCACGGGGACAGGCGTCTGCAGAAGCCTGCGCTCGG
 L  L  T  G  T  G  V  C  R  S  L  R  S 


Exon 7: 3834-4040 (gen)  905-1111 (mRNA)


CTGCCCCCAGGAAATGAGAGTGAGGGTCCACCTGGCTGCCCAGAGGCCCAGCCGCCCCAG
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          GAAATGAGAGTGAGGGTCCACCTGGCTGCCCAGAGGCCCAGCCGCCCCAG
          G  N  E  S  E  G  P  P  G  C  P  E  A  Q  P  P  Q 


GGCCCAGGGCCGGCAGCCTGGGAGGGCTTGTCTGGGGCTGCCACTCCTGCCCCCACTGTG
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
GGCCCAGGGCCGGCAGCCTGGGAGGGCTTGTCTGGGGCTGCCACTCCTGCCCCCACTGTG
 G  P  G  P  A  A  W  E  G  L  S  G  A  A  T  P  A  P  T  V 


CGCCCAGGGACACCGCCAGTGCCCACTCAGCCCACACCTGCAGAGACGAGACTGGAGCCG
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CGCCCAGGGACACCGCCAGTGCCCACTCAGCCCACACCTGCAGAGACGAGACTGGAGCCG
 R  P  G  T  P  P  V  P  T  Q  P  T  P  A  E  T  R  L  E  P 


GCTGCCACCCCCAGGAAGCCCTACACGTGCGAGCAGTGTGGCCGCGG
|||||||||||||||||||||||||||||||||||||
GCTGCCACCCCCAGGAAGCCCTACACGTGCGAGCAGT
 A  A  T  P  R  K  P  Y  T  C  E  Q 


Exon 8: 4314-4338 (gen)  1112-1136 (mRNA)


TGCAGTGACTGTGGCCGCGCCTTCGACTGGAAGTCGCAGCTGGTC
          ||||||||| |||||||||||||||
          GTGGCCGCGGCTTCGACTGGAAGTC
          C  G  R  G  F  D  W  K  S 


Exon 9: 4330-4886 (gen)  1317-1873 (mRNA)


GCGCCTTCGACTGGAAGTCGCAGCTGGTCATCCACCGCAAGGGCCACCGGCCGGAGGTTC
          ||||||||||||||||||||||||||||||| ||||||||||||||||||
          CTGGAAGTCGCAGCTGGTCATCCACCGCAAGAGCCACCGGCCGGAGGTTC

CATGAGCAGCCAGACAGCACAGTCCCTCGGGGCCTCGGTGTTCTCGGGGCCTGGATACAG
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CATGAGCAGCCAGACAGCACAGTCCCTCGGGGCCTCGGTGTTCTCGGGGCCTGGATACAG

CCTCTGGGGCACCAGCAGAAGACTCTGGAGGCAGCAGGGGATGCCAGAGTGAACAAGGGG
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CCTCTGGGGCACCAGCAGAAGACTCTGGAGGCAGCAGGGGATGCCAGAGTGAACAAGGGG

TCCCAAGCCAGTTCCCTGCCCCTGGTCTGGTCTCCCCCAAAAGACCTGGGTGCAAGGAAA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
TCCCAAGCCAGTTCCCTGCCCCTGGTCTGGTCTCCCCCAAAAGACCTGGGTGCAAGGAAA

AGGAGCTGCTCTCTCTCTTCTTGCCCCTGCCTCCTAGAGGGAGGTCTGGGTTCCCTTCTA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
AGGAGCTGCTCTCTCTCTTCTTGCCCCTGCCTCCTAGAGGGAGGTCTGGGTTCCCTTCTA

TGGCTGACCAGTGCCTGTGGGGTGACTGCCAAGCACCAGGCTCCCTCCCTCCCTGTGACA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
TGGCTGACCAGTGCCTGTGGGGTGACTGCCAAGCACCAGGCTCCCTCCCTCCCTGTGACA

TGGCCTGGGCTGACAACACTCCCTCTCCTGGGACCTCCTTGCCTCAGGTGGGTGTTCAAA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
TGGCCTGGGCTGACAACACTCCCTCTCCTGGGACCTCCTTGCCTCAGGTGGGTGTTCAAA

AACTGTGCCTTCCCACTCGTCTGTGCAGAGGCTGGGCCTGAGGTCTCAGTGTGGAGAGCA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
AACTGTGCCTTCCCACTCGTCTGTGCAGAGGCTGGGCCTGAGGTCTCAGTGTGGAGAGCA

GCAGAAGACCCAGGAAAGCACAGTTGGCTTCCGTTTCTCCTGCTCCCTGTGTGTGTTAGA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
GCAGAAGACCCAGGAAAGCACAGTTGGCTTCCGTTTCTCCTGCTCCCTGTGTGTGTTAGA

ATTTTAACATAAATTCCACTTTCATAATATGGAGTTT
|||||||||||||||||||||||||||
ATTTTAACATAAATTCCACTTTCATAA

--SPIDEY version 1.40--
Genomic: lcl|chr19.f0 No definition line found, 4984 bp
mRNA: lcl|BC017206_part No definition line found, 814 bp
Strand: plus
Number of exons: 5
Exon 1: 101-162 (gen)  1-62 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 1  0
Exon 2: 837-1218 (gen)  63-444 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 1  1
Exon 3: 1295-1484 (gen)  445-634 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 1  1
Exon 4: 1745-1839 (gen)  635-729 (mRNA)  id 98.9% mismatches 1 gaps 0  splice site (d  a): 1  1
Exon 5: 3301-3385 (gen)  730-814 (mRNA)  id 100.0% mismatches 0 gaps 0  splice site (d  a): 0  1
Number of splice sites: 4
mRNA coverage: 100%
overall percent identity: 99.9%
Missing mRNA ends: neither 

Genomic: lcl|chr19.f0 No definition line found
mRNA: lcl|BC017206_part No definition line found
Exon 1: 101-162 (gen)  1-62 (mRNA)


GGGACAGATCCCGAAGTTCGAGCATCCCTCGGATAGGCCGGGTGTCAGGCCTGGTCTCTC
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          CCGAAGTTCGAGCATCCCTCGGATAGGCCGGGTGTCAGGCCTGGTCTCTC

AGGCCCGTCCAGGTGGGCGTCT
||||||||||||
AGGCCCGTCCAG

Exon 2: 837-1218 (gen)  63-444 (mRNA)


TTTTCTGTAGGCCCATCTTGACGATTCCAAGACCACCCCCTTGAGCAAGAATGCCATCCC
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          GCCCATCTTGACGATTCCAAGACCACCCCCTTGAGCAAGAATGCCATCCC
                      R  F  Q  D  H  P  L  E  Q  E  C  H  P 


CTCTGGGTCCCCCATGCCTGCCCGTCATGGACCCAGAGACCACCCTTGAGGAGCCTGAGA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CTCTGGGTCCCCCATGCCTGCCCGTCATGGACCCAGAGACCACCCTTGAGGAGCCTGAGA
 L  W  V  P  H  A  C  P  S  W  T  Q  R  P  P  L  R  S  L  R 


CTGCCCGCCTCCGCTTCCGAGGGTTCTGCTACCAGGAGGTGGCAGGTCCCCGAGAAGCCC
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CTGCCCGCCTCCGCTTCCGAGGGTTCTGCTACCAGGAGGTGGCAGGTCCCCGAGAAGCCC
 L  P  A  S  A  S  E  G  S  A  T  R  R  W  Q  V  P  E  K  P 


TGGCCCGGCTGCGTGAGCTGTGTTGCCAGTGGCTGCAGCCTGAGGCACACTCCAAGGAGC
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
TGGCCCGGCTGCGTGAGCTGTGTTGCCAGTGGCTGCAGCCTGAGGCACACTCCAAGGAGC
 W  P  G  C  V  S  C  V  A  S  G  C  S  L  R  H  T  P  R  S 


AGATGCTGGAGATGCTGGTGCTGGAGCAGTTCCTGGGCACACTGCCTCCCGAGATCCAGG
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
AGATGCTGGAGATGCTGGTGCTGGAGCAGTTCCTGGGCACACTGCCTCCCGAGATCCAGG
 R  C  W  R  C  W  C  W  S  S  S  W  A  H  C  L  P  R  S  R 


CCTGGGTGCGCGGTCAGCGGCCAGGCAGTCCTGAGGAGGCCGCTGCCCTAGTCGAAGGAC
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CCTGGGTGCGCGGTCAGCGGCCAGGCAGTCCTGAGGAGGCCGCTGCCCTAGTCGAAGGAC
 P  G  C  A  V  S  G  Q  A  V  L  R  R  P  L  P  * 


TGCAGCATGACCCTGGGCAACTGTTGGGCTGGGTGAGTGTGG
||||||||||||||||||||||||||||||||
TGCAGCATGACCCTGGGCAACTGTTGGGCTGG



Exon 3: 1295-1484 (gen)  445-634 (mRNA)


TTCTCCTCAGATCACAGCCCATGTCCTGAAGCAGGAGGTGCTCCCTGCAGCCCAGAAGAC
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          ATCACAGCCCATGTCCTGAAGCAGGAGGTGCTCCCTGCAGCCCAGAAGAC

AGAGGAACCACTTGGGAGCCCCCACCCCTCAGGGACAGTGGAGTCCCCTGGGGAAGGTCC
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
AGAGGAACCACTTGGGAGCCCCCACCCCTCAGGGACAGTGGAGTCCCCTGGGGAAGGTCC

CCAGGACACCAGAATAGAGGGGTCTGTCCAGCTCAGCTGCAGTGTGAAGGAGGAGCCCAA
||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
CCAGGACACCAGAATAGAGGGGTCTGTCCAGCTCAGCTGCAGTGTGAAGGAGGAGCCCAA

TGTCGATGGACAGGAAGTGGGTGAGGTTGG
||||||||||||||||||||
TGTCGATGGACAGGAAGTGG

Exon 4: 1745-1839 (gen)  635-729 (mRNA)


CACTTTTCAGCACCCTCCAGCCCTCCACTTGCAGCACAGTCCCCTGAGGGGAACCATGGA
          ||||||||||||||||||||||||||||||||||||||||| ||||||||
          CACCCTCCAGCCCTCCACTTGCAGCACAGTCCCCTGAGGGGCACCATGGA

CACCAAGAACCAGCCTCCACATCCTTCCACCCACCCAGGATTCAGGTGAGCAGCC
|||||||||||||||||||||||||||||||||||||||||||||
CACCAAGAACCAGCCTCCACATCCTTCCACCCACCCAGGATTCAG

Exon 5: 3301-3385 (gen)  730-814 (mRNA)


ACATCCGCAGGAGGAGTGGGGGCTGCTGGACCGGTCACAGAAGGAACTGTACTGGGATGC
          ||||||||||||||||||||||||||||||||||||||||||||||||||
          GAGGAGTGGGGGCTGCTGGACCGGTCACAGAAGGAACTGTACTGGGATGC

GATGCTGGAGAAGTACGGCACAGTGGTCTCCCTGGGTGAGGACCA
|||||||||||||||||||||||||||||||||||
GATGCTGGAGAAGTACGGCACAGTGGTCTCCCTGG

