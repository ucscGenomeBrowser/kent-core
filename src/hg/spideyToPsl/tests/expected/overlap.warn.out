Warning: BC017206 + 0-1873  chr19.f0 + 100-4338 has overlapping exons at chr19.f0 4329
